src/management/Akka.Management/Dsl/AkkaManagement.cs

```diff
@@ -32,6 +32,17 @@ public class AkkaManagement : IExtension
         private readonly AtomicReference<Task<ServerBinding>> _bindingFuture = new AtomicReference<Task<ServerBinding>>();
 
         public AkkaManagementSettings Settings { get; }
+        
+        public string HostName
+        {
+            get
+            {
+                var hostname = Settings.Http.Hostname;
+                return hostname.Count(c => c == ':') == 7 ? $"[{hostname}]" : hostname;                
```

Comment:
IPAddress.ToString() returns "2001:db8:85a3:8d3:1319:8a2e:370:7348" instead of "[2001:db8:85a3:8d3:1319:8a2e:370:7348]". This can create problems when you string interpolate it into an URI.